core/src/main/scala/com/pingcap/tispark/TiDBOptions.scala

```diff
@@ -65,7 +65,10 @@ class TiDBOptions(@transient val parameters: CaseInsensitiveMap[String]) extends
   val skipCommitSecondaryKey: Boolean =
     parameters.getOrElse(TIDB_SKIP_COMMIT_SECONDARY_KEY, "true").toBoolean
 
-  val sampleFraction: Double = parameters.getOrElse(TIDB_SAMPLE_FRACTION, "0.01").toDouble
+  val regionSplitNum: Int = parameters.getOrElse(TIDB_REGION_SPLIT_NUM, "3").toInt
+
+  val enableRegionSplit: Boolean =
+    parameters.getOrElse(TIDB_ENABLE_REGION_SPLIT, "false").toBoolean
```

Comment:
suggest to
1.  change `TIDB_ENABLE_REGION_SPLIT` to `TIDB_ENABLE_REGION_PRE_SPLIT` 
2. and change default value from `false` to `true`
3. update document for this parameter

Reply:
`TIDB_ENABLE_REGION_SPLIT` is intended. We actually will do region split as long as the data to be inserted needs to do region split. Per the discussion with engineer from TiDB, we are possible meet hot region problem during writing.